app/assets/javascripts/application.js

```diff
@@ -71,15 +71,20 @@ $(document).ready(function() {
     var message_box = '';
 
     var qty = $('#qty-selected-'+productId).val()
-
     if (action === 'add') {
       //Add a new item to cart
+      $('#qty-selected-'+productId).attr('disabled','disabled');
```

Comment:
aca se podría hacer lo mismo que comente en el PR anterior.

Reply:
Bueno, ahí usé la misma variable que estaba declarada arriba, también la variable that para cambiar el icono del botón. Lo que sí, lo modifiqué todo en esta rama :s